go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aws/amazon-vpc-cni-k8s
 
-go 1.14
+go 1.17
```

Comment:
But still the base image is based of 1.16 - https://github.com/aws/amazon-vpc-cni-k8s/blob/master/Makefile#L61

Reply:
will update that too

Reply:
Can you also please upgrade the go version used in workflow files - https://github.com/aws/amazon-vpc-cni-k8s/blob/master/.github/workflows/build-multi-arch.yml#L18 along with this change ? Thanks